### ISSUE TITLE
View has wrong width

### ISSUE THREAD

**Issue by electrolobzik** [JoanZapata/android-iconify#118]
I have upgraded to v.2.0.8 and got issue on my view:
````
<com.joanzapata.iconify.widget.IconTextView
				android:id="@+id/trashImg"
				android:layout_width="wrap_content"
				android:layout_height="match_parent"
				android:gravity="center"
				android:layout_alignParentRight="true"
				android:layout_alignParentEnd="true"
				android:layout_centerVertical="true"
				android:layout_margin="@dimen/default_padding"
				android:text="{md-delete}"
				android:textColor="@color/white"
				android:textSize="@dimen/big_material_icon_text_size"
				/>
````

In 2.0.2:
![In 2.0.2](http://i.imgur.com/E1o8JMT.png)

In 2.0.8:
![In 2.0.8](http://i.imgur.com/UMpxkDx.png)

**Comment by electrolobzik** [comment]
I have also similar issue with my custom View nested from TextView. I use 
````
setText("{md-check}");
Iconify.addIcons(this);
````
to get the icon in it.

This is the view in layout:
````
<net.privacyeverywhere.ui.view.CircleLetterView
android:id="@+id/mailRoundButtonTxt"
android:layout_width="@dimen/mail_item_circle_width"
android:layout_height="@dimen/mail_item_circle_width"
android:layout_centerVertical="true"
android:layout_marginRight="@dimen/small_padding"
android:layout_marginEnd="@dimen/small_padding"
/>
````

In 2.0.2:
![2.0.2](http://i.imgur.com/bXVRcgx.png)

In 2.0.8:
![2.0.8](http://i.imgur.com/1iLUagK.png)

**Comment by JoanZapata** [comment]
Thank you so much for all the details, I think I know what it is. Between version 2.0.2 and 2.0.8 I changed the way the icon are rendered. In 2.0.8 I compute their size and draw them myself on the canvas (to be able to rotate them when the user is using `spin`). I'll probably need to tweak [those values](https://github.com/JoanZapata/android-iconify/blob/master/android-iconify/src/main/java/com/joanzapata/iconify/internal/CustomTypefaceSpan.java#L50-L53) a little bit. I'll try to get some time to get it right before the end of the week.

**Comment by JoanZapata** [comment]
The fix will be published on 2.0.10 soon
